src/components/schemas/category.yaml

```diff
@@ -75,6 +75,10 @@ category_out_embed:
       description: Represents the name of the sub_category.
     code:
       $ref: ./fields.yaml#/code
+    display_name:
+      type: string 
+      example: Engine / Turbo charged 
+      description: Represents the display name of the category i.e category-name/sub_category-name
```

Comment:
Same as below

Reply:
In the display_name description in the `category_out`. 
We have added this line `In case if no sub_category is present it is same as name of the category.` to display name. Maybe we can add the same here too